src/main/kotlin/com/exactpro/th2/sailfish/utils/transport/TransportToIMessageConverter.kt

```diff
@@ -253,8 +254,13 @@ class TransportToIMessageConverter @JvmOverloads constructor(
     }
 
     companion object {
+        private const val UNKNOWN_NAMESPACE = "unknown"
         private val K_LOGGER = KotlinLogging.logger {}
         private val CONVERTERS = initConverters()
+
+        @JvmStatic
+        val DEFAULT_MESSAGE_FACTORY: IMessageFactory = DefaultMessageFactory.getFactory()
```

Comment:
Please, use @JvmField instead
```suggestion
        @JvmField
        val DEFAULT_MESSAGE_FACTORY: IMessageFactory = DefaultMessageFactory.getFactory()
```